### PR TITLE
Tagliatelle recipe

### PR DESCRIPTION
**Pull request by dineshraj**

### CODE REVIEW

File: _posts/2021-03-09-one-pot-tagliatelle-with-olives-capers-and-sundried-tomatoes.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
+---
+layout: post
+title:  One Pot Tagliatelle with Olives Capers and Sundried Tomatoes Recipe
```
Comment (drairi) [inline _posts/2021-03-09-one-pot-tagliatelle-with-olives-capers-and-sundried-tomatoes.md:3]: Add a comma after Olives

File: _posts/2021-03-09-one-pot-tagliatelle-with-olives-capers-and-sundried-tomatoes.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+* Add the 200g of tagliatelle.
+
+* Cover the tagliatelle with the passata.
+
+* Add the water and use a wooden spoon to push down and crack the tagliatelle if it is not fully submerged.
+
+* Bring to the boil and then reduce to a simmer.
+
+{% include important.html text='<strong>Note:</strong> If using a stainless steel pan (or if your non-stick has worn off) keep stiring to ensure the pasta does not stick to the bottom of the pan' %}
```
Comment (drairi) [inline _posts/2021-03-09-one-pot-tagliatelle-with-olives-capers-and-sundried-tomatoes.md:58]: "stirring" has two r's

File: _posts/2021-03-09-one-pot-tagliatelle-with-olives-capers-and-sundried-tomatoes.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+* Add the sugar, capers, olives, sundried tomatoes, chorizo (optional), chillis and the remaining 2 tbsp of olive oil.
+
+* Stir well and cover the pain with the lid, cook for roughly 10 mins (until the pasta is done), and stir regularly to prevent sticking.
```
Comment (drairi) [inline _posts/2021-03-09-one-pot-tagliatelle-with-olives-capers-and-sundried-tomatoes.md:62]: cover the pan 🥘 , not pain 🥖

**Review by drairi — CHANGES_REQUESTED** [review]
Looks great! Just 3 changes required.

File: _posts/2021-03-09-one-pot-tagliatelle-with-olives-capers-and-sundried-tomatoes.md (hunk 3 above)
Comment (dineshraj) [reply _posts/2021-03-09-one-pot-tagliatelle-with-olives-capers-and-sundried-tomatoes.md:62]: pain ☠︎

**Pull request merged**